src/app/frontend/deploy/helpsection/userhelp.scss

```diff
@@ -11,22 +11,24 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-@import "../../../assets/styles/color_variables";
+@import '../../../assets/styles/color_variables';
 
-.kd-user-help * {
-  color: $muted;
-}
+.kd-user-help {
+  * {
+    color: $muted;
+  }
 
-.kd-user-help a {
-  color: $primary;
-  opacity: 1;
-  text-decoration: none;
-}
+  a {
+    color: $primary;
+    opacity: 1;
+    text-decoration: none;
 
-.kd-user-help b {
-  color: black;
-}
+    :hover {
```

Comment:
Thank you for pointing this. Indeed, I made mistake. The correct syntax is: `&:hover`. Fixed. Source: http://sass-lang.com/documentation/file.INDENTED_SYNTAX.html